ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/events/AlterTableHandler.java

```diff
@@ -135,8 +173,15 @@ private boolean handleForTableLevelReplication(Context withinContext) {
         return false;
```

Comment:
In fact, all the flows in this else path returns false. So, shall remove this return statement.

Reply:
just for code maintainability the checks are kept.

Reply:
Same as above.

Reply:
As discussed, let's keep the checks.